specification/deviceprovisioningservices/resource-manager/readme.md

```diff
@@ -26,19 +26,28 @@ These are the global settings for the API.
 
 ``` yaml
 openapi-type: arm
-azure-arm: true 
-tag: package-2022-02
+azure-arm: true
+tag: package-preview-2022-08
 ```
 
 
+### Tag: package-preview-2022-08
+
+These settings apply only when `--tag=package-preview-2022-08` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2022-08'
+input-file:
+  - Microsoft.Devices/preview/2022-08-07-preview/iotdps.json
+```
 ### Tag: package-2022-02
 
 These settings apply only when `--tag=package-2022-02` is specified on the command line.
 
-```yaml $(tag) == 'package-2022-02'
+``` yaml $(tag) == 'package-2022-02'
```

Comment:
I recommend that you revert the change on this line. Maybe this will fix the whacky PR checks.
```suggestion
```yaml $(tag) == 'package-2022-02'
```